.github/CHANGELOG.md

```diff
@@ -124,6 +124,7 @@
 1. [FMS] Run vertical predictions without V-speeds - @BlueberryKing (BlueberryKing)
 1. [ELEC] Use ADIRU and LGCIU signals for speed and in flight determination - @Gurgel100 (Pascal)
 1. [FMS] Do not transmit bearing information to ND on manual legs - @BravoMike99 (bruno_pt99)
+1. [MCDU] Suppress TMPY FPLN when no changes made on airways page - @robertxing2004 (robeet)
```

Comment:
Wrong version, needs to be under 0.13.0 (you might need to rebase/merge master first).

Reply:
Ah yes, you're right.